### PR TITLE
[REV] pivot: select pivot cells in composer

### PR DESCRIPTION
**Pull request by LucasLefevre**
This reverts commit 6c4a8e75631c41fd49fc297e995d95969c6060f3.

It is quite annoying to build a formula by referring a pivot value and getting the full pivot formula rather than its cell address.

So a good behavior IMO would be:

If you copy a value from a pivot and paste it somewhere else, you would get the full formula (i.e. =PIVOT(1,$$$,$$$,$$$))
If you select a value from a pivot while building a formula within the composer, you would get its address (i.e. =SUM(1+C3+4)) and not (=SUM(1+(PIVOT(1,$$$,$$$,$$$))+4)

Additional notes:

With the current behavior (full formula inserted) and if you actually want the reference, it's very annoying: you have to delete the full formula, carefully check the reference of the cell you want to target, then type the reference by hand. The other way around is slightly easier: either build the formula using the auto-complete assistant, or copy-paste the cell somewhere else and reference that cell with the fixed formula.

There are pros & cons with both behaviors, and they are incompatible. I say: if you use a dynamic pivot, we let you embrace dynamic all the way, otherwise just use a fixed formula pivot. Reverting full formula insertion is also less code and less surprising (not a different behavior just for pivots)

Task: 4189098

## Description:

description of this task, what is implemented and why it is implemented that way.

Task: [TASK_ID](https://www.odoo.com/web#id=TASK_ID&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## review checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_t("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

File: src/helpers/rendering.ts [hunk 1]
```diff
@@ -16,7 +16,6 @@ export function drawHighlight(
   const color = highlight.color || HIGHLIGHT_COLOR;
 
   const { ctx } = renderingContext;
-  ctx.save();
```
Comment (hokolomopo) [inline src/helpers/rendering.ts:19]: why is this removed ?

**Review by hokolomopo — COMMENTED** [review]
+1, having formulas being automatically written when selecting a cell is really annoying IMO. Excel does it a lot with tables, and I hate it :imp:

File: src/helpers/rendering.ts (hunk 1 above)
Comment (LucasLefevre) [reply src/helpers/rendering.ts:19]: because it was added in the reverted commit, but I don't remember why :man_shrugging: 
It was probably a "bug" that only appeared in this case. I'll restore it

**Comment by rrahir** [comment]
@robodoo r+
